public/app/pages/ContinuousDiffView.tsx

```diff
@@ -218,7 +220,7 @@ function ComparisonDiffApp() {
             />
           </Panel>
         </div>
-        <Panel isLoading={isLoading} title={<ChartTitle titleKey="diff" />}>
```

Comment:
Changed due to TS issues. We actually don't really need it in the UI. It saves a bit of vertical real estate (more above the search input):

<img width="1470" alt="image" src="https://github.com/grafana/pyroscope/assets/146180665/3f9bcdd7-646c-44ad-801c-0acbfe7c834a">